scenario/utils/relayArbitrumMessage.ts

```diff
@@ -81,6 +82,25 @@ export async function relayArbitrumMessage(
       bridgeDeploymentManager,
       sender
     );
+    // if method name == finalizeInboundTransfer(address,address,address,uint256,bytes)
+    if(data.slice(0, 10) == '0x2e567b36'){
+      const _data = '0x' + data.slice(10, 266);
+      const [token,, to, amount] = utils.defaultAbiCoder.decode(
+        ['address', 'address', 'address', 'uint256'],
+        _data
+      );
+      // if token is mainnet ETH -> than source arbitrum weth
```

Comment:
Simulate Arbitrum bridge WETH main net -> WETH arbitral bridging. Without it, the transaction will fail here, because `from: sender`. It should be an Arbitrum Bridge contract. https://github.com/compound-finance/comet/pull/860/files#diff-a36faddb9a40a7866ee100dcc5695b5c86c8de1dde25bd1e74d60fc73ab66be7R104